### ISSUE TITLE
Visible stepping behaviour in recent versions reports values

### ISSUE THREAD

**Issue by DavidGasku** [jmoenig/Snap#3179]
I guess this is an intentional change to debug scripts. 
But I usually use Stepping at its fastest speed to help the students learn the execution flow.
Would be nice to make the reporter behaviour an independent toggle in the UI, as now every custom block is reporting lots of stuff.

**Comment by jmoenig** [comment]
yes, that's intentional and requested by the UCB folks. Note that it only reports intermediate value for scripts that get triggered by directly clicking on them.

**Comment by DavidGasku** [comment]
Ok, just wanted to suggest the possibility of making the reporting and highlighting two separate options.

**Comment by jmoenig** [comment]
Thanks! Will bring it up with @ddgarcia at the next meeting.

**Comment by cycomachead** [comment]
Yeah, we should discuss this - i think there's probably a middle ground of "slow enough you can actually read the output" so it shows only on fairly slow stepping settings.

**Comment by jmoenig** [comment]
right, but this would make the feature way less discoverable. @DavidGasku I like to understand how showing values in visible stepping is causing an issue for what sounds like a great practice in your classroom. Are the bubbles blocking the view?

**Comment by DavidGasku** [comment]
No, it's just distracting. In some activities they use blocks that I prepared for them, and the intermediate reports from those blocks are not relevant for them. It would be fine if warp would silence those, but it is not the case
